public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -290,11 +291,17 @@ export class CloudWatchDatasource
     metricQueries: CloudWatchMetricsQuery[],
     options: DataQueryRequest<CloudWatchQuery>
   ): Observable<DataQueryResponse> => {
+    const timezoneUTCOffset = dateTimeFormat(Date.now(), {
+      timeZone: options.timezone,
+      format: 'Z',
+    }).replace(':', '');
```

Comment:
Looking at the [moment tz docs](https://momentjs.com/timezone/docs/#/using-timezones/formatting/), it seems like `format: 'z'` would return timezone offset without `:`. But when I used lower case `z` it didn't return any value. :thinking: